pkg/clients/ec2/routetable.go

```diff
@@ -198,6 +206,7 @@ func IsRtUpToDate(p v1beta1.RouteTableParameters, rt ec2types.RouteTable) (bool,
 	return cmp.Equal(&v1beta1.RouteTableParameters{}, patch,
```

Comment:
Does that make sure to not check the routes?

Reply:
added checks in CreateRTPatch to skip routes - does this fits ? or did you want this on a other code block ?

Reply:
@muvaf did you want to have the check earlier ?